### PR TITLE
Update of CRT Hit Reconstruction: Top CRT TS0 corrected for fiber pro…

### PR DESCRIPTION
**Pull request by francescopoppi**
…pagation, TS1 included in HitTree, TS0/TS1 corrected SiPM to FEB delay, Dummy Gain/Pedestal for Top CRT included in fcl files (crtsimhitproducer.fcl)

### CODE REVIEW

File: icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc [hunk 1]
```diff
@@ -364,19 +367,28 @@ sbn::crt::CRTHit CRTHitRecoAlg::MakeTopHit(art::Ptr<CRTData> data){
...
 
-    if(adsid_max<8)
-        thit -= hitpos.Z()*fPropDelay;
-    else
-        thit -= hitpos.X()*fPropDelay;
-
+    if(adsid_max<8){
+        thit -= (uint64_t)round(abs((92+hitpos.X())*fPropDelay));
+        thit1 -= (uint64_t)round(abs((92+hitpos.X())*fPropDelay));
+	thit -= 11; //Correction for 11 ns signal cable from SiPM to FEB
```
Comment (biswaranjan91) [inline icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc:375]: Thanks for putting this together.
Please add a fhicl parameter for 11 ns. It is always better to remove any hardcoded information. Same comments for L381 and L382.

**Review by biswaranjan91 — COMMENTED** [review]
Thanks for putting together. 
Seems we missed to push the file for your changes mentioned ☝️ (TS1 included in HitTree).
Also we need to push the same changes (file) to the production branch SBN2022A. Thanks.

File: icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc (hunk 1 above)
Comment (francescopoppi) [reply icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc]: @biswaranjan91 added the SiPM to FEB propagation as crtsimproducer.fcl parameter

**Comment by rennney** [comment]
trigger build

**Comment by rennney** [comment]
trigger build

**Comment by rennney** [comment]
trigger build

**Comment by rennney** [comment]
trigger build

**Pull request merged**